data/coalition/lunarium intro.txt

```diff
@@ -1118,6 +1118,18 @@ mission "Lunarium: Evacuation 5"
 				accept
 	on visit
 		dialog `You land on <planet>, but realize that your escort with the <bunks> bunks for the immigrants hasn't arrived in the system yet. Better depart and wait for it.`
+	on fail
+		conversation
+			`You're contacted by Elirom upon landing.`
```

Comment:
Normally I'm not big on contradictions, but I feel as a first line for an important conversation, it should probably be "You are" for more of a stern impact, if that makes sense.